File: elias/Problema0377/problema0377.py
matriz = []
somaElementosDiagonalPrincipal = 0
for i in range(3):
    linhaMatriz = []
    for j in range(3):
        x = int(input(f"Digite um número para a linha {i} e coluna {j} da matriz:")) 
        linhaMatriz.append(x)
    matriz.append(linhaMatriz)
for i in range(3):
    for j in range(3):
        if i == j:
            somaElementosDiagonalPrincipal = somaElementosDiagonalPrincipal + matriz[i][j]
mediaElementosDiagonalPrincipal = somaElementosDiagonalPrincipal / 3
print("A média dos elementos da diagonal principal é: ",mediaElementosDiagonalPrincipal)

File: elias/Problema0377/problema0377.java
import java.util.Scanner;

public class problema0377 {
    public static void main(String[] args) {
        Scanner input = new Scanner (System.in);
        int[][] matriz = new int[3][3];
        int somaElementosDiagonalPrincipal = 0;
        float mediaElementosDiagonalPrincipal;
        for(int i=0;i<3;i++){
            for(int j=0;j<3;j++){
                System.out.println("Digite um número para a linha "+i+" e coluna "+j+" :");
                matriz[i][j] = Integer.valueOf(input.nextLine()); 
            }
        }
        for(int i=0;i<3;i++){
            for(int j=0;j<3;j++){
                if(i == j){
                    somaElementosDiagonalPrincipal = somaElementosDiagonalPrincipal + matriz[i][j];
                }
            }
        }
        mediaElementosDiagonalPrincipal = somaElementosDiagonalPrincipal / 3;
        System.out.println("A média dos elementos da diagonal principal é: "+mediaElementosDiagonalPrincipal);
    }
}